release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -274,6 +274,17 @@ The `openshiftVersion` parameter of the `AgentServiceConfig` custom resource (CR
 
 For more information see, xref:../scalability_and_performance/routing-optimization.adoc#configuring-ingress-controller-router-liveness-readiness-and-startup-probes[Configuring Ingress Controller (router) Liveness, Readiness, and Startup probes]
 
+[id=ocp-4-11-node-observability-operator]
+==== Node Observability Operator (Technology Preview)
+{product-title} 4.11 introduces the Node Observability Operator in Technology Preview.
+
+Node Observability Operator provides the following features:
+
+* Trigger a CRIO-O and Kubelet profiling.
+* Deploy Node Observability Agents on selected nodes.
+
```

Comment:
I think that the third phrase could be kept but rephrased to something like:

> Make the profiling data files available for the further analysis.

It's not quite a feature, but the deployment of the agent is not either.
What we are talking about here is more a mix of the features and the operator's flow.
With the third phrase: the operator's flow would be logically complete.

Reply:
makes sense. I will add this point.